Domain reorg software

iankirkman opened the pull request.

k8hertweck commented on _bioinfcomputing/software_languages.md:
```diff
@@ -0,0 +1,43 @@
+
+## Training Resources 
+
+Fred Hutch offers training resources for each of the common languages and tools listed below. For course descriptions and session notifications, go to Hutch Learning [here](http://www.fredhutch.io/). New class offerings will also be posted in the Coop Newsletter. Email `thecoop` to join the mailing list.
```
I'd revise this section as follows:

Fred Hutch offers training resources for many of the common languages and tools listed below. Course descriptions can be found at [fredhutch.io](http://www.fredhutch.io/), and course registration is tracked in Hutch Learning (only accessible to Fred Hutch employees through CenterNet; search for "fredhutch.io" to see all relevant courses). New class offerings will also be posted in the Coop Newsletter. Email `thecoop@fredhutch.org` to join the mailing list.

iankirkman replied:
Updated except for the email address, because the wiki convention is to hide the details.

k8hertweck commented on _bioinfcomputing/software_overview.md:
```diff
@@ -0,0 +1,61 @@
+
+Fred Hutch has several resources available for folks that are seeking instruction in a new programming language, version control (managing and sharing code), or navigating linux/shell scripting. If software development in research is new to you, or you wish to pick up a new skill, we recommend you start with the [Common Languages](/bioinfcomputing/software_languages/) page.
+
+The remainder of this section will presume a basic skill level in at least one common language, as well as a basic understanding of [GitHub](/bioinfcomputing/software_managecode/).
```
I think this does a great job accomplishing the goal we discussed previously: managing expectations for both experienced users and newbies.

k8hertweck commented on _bioinfcomputing/software_overview.md:
```diff
@@ -0,0 +1,61 @@
+
+
+### Submitting Your Code
+When you have completed a coding project that you would like to submit to the examples wiki, there are a few steps in Github to navigate the review process. You will want to push your code in a clean branch to the repository, and then submit a pull request on `github.com` for your branch that will kick off a review and merge process.
```
I personally prefer instructions for submission to be kept within the repo itself. If we do want to keep this here, there should be a description to how to get added to the GitHub org so it's possible to create a branch within the original repo.

iankirkman commented:
@k8hertweck I've addressed all of your comments and I think we're ready to merge. I'm also curious if you've taken a look at the new templates repo? https://github.com/FredHutch/wiki-templates
